### ISSUE TITLE
Configuring with TCP fails to run container

### ISSUE THREAD

**Issue by arashm** [kylemanna/docker-openvpn#175]
I used the TCP commands from docs on a Ubuntu 16.04 (digital ocean server) as follows:

```
docker run -v ovpn-data:/etc/openvpn --rm kylemanna/openvpn ovpn_genconfig -u tcp://some.ip.address.here:443
docker run -v ovpn-data:/etc/openvpn --rm -it kylemanna/openvpn ovpn_initpki
```

but when I run

```
docker run -v ovpn-data:/etc/openvpn --rm -p 443:1194/tcp --cap-add=NET_ADMIN kylemanna/openvpn
```

I get:

```
/usr/local/bin/ovpn_run: line 40: /etc/openvpn/ovpn_env.sh: No such file or directory
```

I tried removing images and ovpn-data container to start fresh, but it didn't change the result and I get the same error every time.

**Comment by kylemanna** [comment]
I'm wondering if some versions of Docker aren't creating the volume as expected.

Can you give me the version info from `docker info`  and try running

    docker volume create --name ovpn-data

Before creating the volume to see if manually creating the volume helps?

**Comment by arashm** [comment]
That fixed the issue. I close the issue.
Here's the `docker info` anyway:

```
Containers: 3
 Running: 2
 Paused: 0
 Stopped: 1
Images: 3
Server Version: 1.12.3
Storage Driver: devicemapper
 Pool Name: docker-253:1-652989-pool
 Pool Blocksize: 65.54 kB
 Base Device Size: 10.74 GB
 Backing Filesystem: xfs
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 100.8 MB
 Data Space Total: 107.4 GB
 Data Space Available: 18.88 GB
 Metadata Space Used: 749.6 kB
 Metadata Space Total: 2.147 GB
 Metadata Space Available: 2.147 GB
 Thin Pool Minimum Free Space: 10.74 GB
 Udev Sync Supported: true
 Deferred Removal Enabled: false
 Deferred Deletion Enabled: false
 Deferred Deleted Device Count: 0
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 WARNING: Usage of loopback devices is strongly discouraged for production use. Use `--storage-opt dm.thinpooldev` to specify a custom block storage device.
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.110 (2015-10-30)
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge overlay null host
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Security Options: apparmor seccomp
Kernel Version: 4.4.0-45-generic
Operating System: Ubuntu 16.04.1 LTS
OSType: linux
Architecture: x86_64
CPUs: 1
Total Memory: 488.5 MiB
Name: merlyn
ID: E7T6:BXVX:P53Z:AWPM:RYFK:5ORY:E5B4:X4SE:JJYN:ZFQH:LKWZ:M42E
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: No swap limit support
Insecure Registries:
 127.0.0.0/8
```

**Comment by kylemanna** [comment]
@arashm I'm wondering if I should add that explicit step to the README file.  I think I'll add it to be explicit to those following the guide.

The current Docker documentation [Mount a shared-storage volume as a data volume](https://docs.docker.com/engine/tutorials/dockervolumes/#/mount-a-shared-storage-volume-as-a-data-volume) suggests that the volume should be auto created.
